src/assets/less/fonts.less

```diff
@@ -0,0 +1,57 @@
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Avenir (body) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@font-face {
+    font-family: 'AvenirNext';
+    font-style : normal;
+    font-weight: normal; /* 400, Regular */
+    src        : url("https://fonts.particle.io/AvenirNext/721263/2cd55546-ec00-4af9-aeca-4a3cd186da53.woff2") format("woff2"),
+                 url("https://fonts.particle.io/AvenirNext/721263/1e9892c0-6927-4412-9874-1b82801ba47a.woff") format("woff"),
+                 url("https://fonts.particle.io/AvenirNext/721263/46cf1067-688d-4aab-b0f7-bd942af6efd8.ttf") format("truetype");
+}
+@font-face {
+    font-family: 'AvenirNext';
+    font-style : normal;
+    font-weight: 500; /* Medium */
+    src        : url("https://fonts.particle.io/AvenirNext/721275/627fbb5a-3bae-4cd9-b617-2f923e29d55e.woff2") format("woff2"),
+                 url("https://fonts.particle.io/AvenirNext/721275/f26faddb-86cc-4477-a253-1e1287684336.woff") format("woff"),
+                 url("https://fonts.particle.io/AvenirNext/721275/63a74598-733c-4d0c-bd91-b01bffcd6e69.ttf") format("truetype");
+}
+@font-face {
+    font-family: 'AvenirNext';
+    font-style : normal;
+    font-weight: 600; /* Demi Bold */
+    src        : url("https://fonts.particle.io/AvenirNext/721269/aad99a1f-7917-4dd6-bbb5-b07cedbff64f.woff2") format("woff2"),
+                 url("https://fonts.particle.io/AvenirNext/721269/91b50bbb-9aa1-4d54-9159-ec6f19d14a7c.woff") format("woff"),
+                 url("https://fonts.particle.io/AvenirNext/721269/a0f4c2f9-8a42-4786-ad00-fce42b57b148.ttf") format("truetype");
+}
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ DIN (headings) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@font-face {
+    font-family: 'DIN W01 Cond Bold';
+    font-style : normal;
+    font-weight: bold; /* 700, Bold */
+    src        : url("https://fonts.particle.io/f9fb0794-9b57-4d8c-99f6-69d45d311f29.woff2") format('woff2'),
+                 url("https://fonts.particle.io/ac30a449-22f6-441a-a3cd-93251696660e.woff") format('woff'),
+                 url("https://fonts.particle.io/1c234575-4907-46fb-a87c-4ab80464bb84.ttf") format('truetype');
+}
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Fira Mono (monospaced) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@font-face {
+    font-family: 'Fira Mono';
+    font-style : normal;
+    font-weight: 500; /* Medium */
+    src        : url("https://s3.amazonaws.com/fonts.particle.io/FiraMono-Medium.woff2") format('woff2'),
```

Comment:
Can you switch all the FiraMono URLs to fonts.particle.io (like https://fonts.particle.io/FiraMono-Medium.woff2) to take advantage of caching provided by Cloudfront?